### PR TITLE
Adding missing check mode to info modules

### LINKED ISSUE #139: All _info/_facts modules must work in check_mode
##### SUMMARY
There are multiple _info/_facts modules in this collection which do not support check_mode.

https://github.com/ansible/ansible/pull/75324:
```
ERROR: plugins/modules/digital_ocean_account_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_balance_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_certificate_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_database_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: not a valid value for dictionary value @ data['supports_check_mode']. Got False
ERROR: plugins/modules/digital_ocean_domain_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_droplet_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_firewall_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_floating_ip_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_image_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_kubernetes_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_load_balancer_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_project_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_region_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_size_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_snapshot_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_tag_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
ERROR: plugins/modules/digital_ocean_volume_info.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.supports_check_mode: required key not provided @ data['supports_check_mode']. Got None
```

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
plugins/modules/digital_ocean_account_info.py
plugins/modules/digital_ocean_balance_info.py
plugins/modules/digital_ocean_certificate_info.py
plugins/modules/digital_ocean_database_info.py
plugins/modules/digital_ocean_domain_info.py
plugins/modules/digital_ocean_droplet_info.py
plugins/modules/digital_ocean_firewall_info.py
plugins/modules/digital_ocean_floating_ip_info.py
plugins/modules/digital_ocean_image_info.py
plugins/modules/digital_ocean_kubernetes_info.py
plugins/modules/digital_ocean_load_balancer_info.py
plugins/modules/digital_ocean_project_info.py
plugins/modules/digital_ocean_region_info.py
plugins/modules/digital_ocean_size_info.py
plugins/modules/digital_ocean_snapshot_info.py
plugins/modules/digital_ocean_tag_info.py
plugins/modules/digital_ocean_volume_info.py

##### COLLECTION VERSION
Latest development branch

### PR DESCRIPTION
**Pull request by mamercad**
##### SUMMARY

Fixes #139.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

Bunch of info modules were missing this. Since info modules *shouldn't* actually do anything, I'm wondering if there's anything that actually needs implementation.

### CODE REVIEW

File: plugins/modules/digital_ocean_droplet_info.py [hunk 1]
```diff
@@ -241,7 +241,8 @@ def main():
     argument_spec = DigitalOceanHelper.digital_ocean_argument_spec()
     argument_spec.update(
         name=dict(type='str', required=False, default=None),
-        id=dict(type='str', required=False, default=None)
+        id=dict(type='str', required=False, default=None),
+        supports_check_mode=True
```
Comment (felixfontein) [inline plugins/modules/digital_ocean_droplet_info.py:245]: This needs to go a couple of lines below into `AnsibleModule()`.

File: plugins/modules/digital_ocean_kubernetes_info.py [hunk 2]
```diff
@@ -216,7 +216,8 @@ def main():
                 required=True,
             ),
             name=dict(type='str', required=True),
-            return_kubeconfig=dict(type='bool', default=False)
+            return_kubeconfig=dict(type='bool', default=False),
+            supports_check_mode=True
```
Comment (felixfontein) [inline plugins/modules/digital_ocean_kubernetes_info.py:220]: This needs to go after the next line, right now it's part of the argument spec.

**Review by felixfontein — COMMENTED** [review]
(I guess these two are also what makes the 2.10 sanity tests fail.)

**Comment by felixfontein** [comment]
> Since info modules _shouldn't_ actually do anything, I'm wondering if there's anything that actually needs implementation.

If they don't really change things (I haven't checked), this is enough.

**Comment by mamercad** [comment]
> > Since info modules _shouldn't_ actually do anything, I'm wondering if there's anything that actually needs implementation.
> 
> If they don't really change things (I haven't checked), this is enough.

I checked real quick, all of the API calls are GETs.

**Review by felixfontein — APPROVED** [review]

**Review by mpontillo — APPROVED** [review]

**Pull request merged**